darwin/options.py

```diff
@@ -83,41 +85,56 @@ def __init__(self):
             help="Frames per second for video split (recommended: 1).",
         )
 
+        # Remove
         parser_remove = dataset_action.add_parser(
             "remove", help="Remove a remote or remote and local dataset."
         )
         parser_remove.add_argument("dataset", type=str, help="Remote dataset name to delete.")
 
+        # Report
         parser_report = dataset_action.add_parser("report", help="Report about the annotators ")
         parser_report.add_argument("dataset", type=str, help="Remote dataset name to report on.")
-
         parser_report.add_argument(
             "-g",
             "--granularity",
             choices=["day", "week", "month", "total"],
             help="Granularity of the report",
         )
 
+        # Export
+        parser_export = dataset_action.add_parser(
+            "export", help="Export the a version of a dataset."
+        )
+        parser_export.add_argument("dataset", type=str, help="Remote dataset name to export.")
+        parser_export.add_argument(
+            "annotation_class",
+            type=str,
+            nargs='?',
+            help="List of class filters"
+        )
+        parser_export.add_argument("name", type=str, help="Name with with the version gets tagged.")
```

Comment:
How about putting the name first, since the classes are optional

Reply:
done